.travis.yml

```diff
@@ -6,8 +10,9 @@ before_install:
   - tar xf phantomjs-1.9.7-linux-x86_64.tar.bz2
   - export PATH=$PATH:phantomjs-1.9.7-linux-x86_64/bin/
 install:
-  - bundle install --without production --retry=6
+  - bundle install
```

Comment:
Does this explicit call to `bundle install` conflict with the `bundler_args` below?